keras/layers/convolutional.py

```diff
@@ -379,6 +379,95 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
+class TransposedConvolution2D(Convolution2D):
```

Comment:
keras.backed also says deconv2d. Also, deconv is the most popular term, although not the most precise